### PR TITLE
Add Cattle Bity

### PR DESCRIPTION
**Pull request by mradionov**
Clone of Battle City (1985) by Namco

### CODE REVIEW

File: games/c.yaml [hunk 1]
```diff
@@ -218,6 +218,25 @@
...
+  originals:
+  - Battle City
+  repo: https://github.com/dogballs/cattle-bity
+  url: https://dogballs.github.io/cattle-bity/
+  development: complete
+  status: playable
+  lang:
+  - TypeScript
+  license:
+  - MIT
```
Comment (cxong) [inline games/c.yaml:232]:
```suggestion
  - MIT
content: commercial
```

File: games/c.yaml [hunk 2]
```diff
@@ -230,6 +230,7 @@
   - TypeScript
   license:
   - MIT
+content: commercial
```
Comment (cxong) [inline games/c.yaml]:
```suggestion
  content: commercial
```
oops; I think the indentation is wrong

Comment (mradionov) [reply games/c.yaml]: It should be ok in PR itself, I fixed it

**Comment by wonea** [comment]
That's a fun game!

**Comment by mradionov** [comment]
Thanks!

**Pull request merged**